src/stactools/landsat/utils.py

```diff
@@ -134,3 +149,69 @@ def recursive_round(coordinates: List[Any], precision: int) -> List[Any]:
         item.bbox = recursive_round(list(item.bbox), precision)
 
     return item
+
+
+def update_geometry(
```

Comment:
```suggestion
def _update_geometry(
```

Is there any reason to add this function to the public API? If not, should be made "private", as it's got a pretty complex signature.

Reply:
Nah. Underscore added.